packages/client/src/client.js

```diff
@@ -292,6 +296,11 @@ export default class PercyClient {
     this.addClientInfo(clientInfo);
     this.addEnvironmentInfo(environmentInfo);
 
+    if (!this.envWarning && (this.clientInfo.size === 0 || this.environmentInfo.size === 0)) {
+      this.envWarning = true;
+      this.log.warn('Warning: Missing `clientInfo` and/or `environmentInfo` properties');
+    }
```

Comment:
Small nit
```suggestion
    if (!this.envWarning && (!this.clientInfo.size || !this.environmentInfo.size)) {
      this.log.warn('Warning: Missing `clientInfo` and/or `environmentInfo` properties');
      this.envWarning = true;
    }
```

AND/OR, how annoying do we want this warning / what do we think of logging this for every snapshot (getting rid of the `envWarning` variable)? I'm wondering how easy this might be to miss if only logged once (plus this shouldn't be an issue for our own SDKs (and if so, we need to fix them anyway))